test/dlc_tests/ecs/mxnet/inference/test_ecs_mxnet_inference.py

```diff
@@ -66,6 +69,7 @@ def test_ecs_mxnet_inference_gluonnlp_cpu(
         ecs_utils.tear_down_ecs_inference_service(ecs_cluster_arn, service_name, task_family, revision)
 
 
+@pytest.mark.model("bert_sst")
```

Comment:
nit: gluonnlp is in the function name, but let's still add it as integration for clarity (and in case the function name changes)

Reply:
added